js/coinex.js

```diff
@@ -30,6 +30,7 @@ module.exports = class coinex extends Exchange {
                 'createOrder': true,
                 'createReduceOnlyOrder': true,
                 'fetchBalance': true,
+                'fetchBorrowInterest': true,
```

Comment:
Should probably set `fetchBorrowRates` to false also, or implement it

Reply:
It's already implemented and set to true